contracts/api-spec.yaml

```diff
@@ -613,6 +650,82 @@ components:
           format: int64
           example: '2'
 
+    StreamStatModel:
+      description: A stat that is related to a specific stream
+      properties:
+        id:
+          type: integer
+          format: int64
```

Comment:
Some of these properties can have decimal values, so integer type won't work for them

* link rtt
* link bandwidth
* send mbitRate
* recv mbitRate

Reply:
fixed